src/webapp/components/QuadrantMap.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import ReactMapboxGl, { Feature, Layer, Source } from "react-mapbox-gl";
+import quadrantsGeoJSON from "../../assets/crownheights.json";
+
+const CROWN_HEIGHTS_CENTER_COORD = [-73.943018, 40.671254];
+
+const MapboxMap = ReactMapboxGl({
```

Comment:
CRA fetches this from the env during build time: https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
sure can do